dask_expr/io/tests/test_io.py

```diff
@@ -1,20 +1,25 @@
+import importlib
 import os
 
 import dask.dataframe as dd
-import pandas as pd
 import pytest
+from dask import config
 from dask.dataframe.utils import assert_eq
 
 from dask_expr import from_dask_dataframe, from_pandas, optimize, read_csv, read_parquet
 from dask_expr._expr import Expr, Lengths, Literal, Replace
 from dask_expr._reductions import Len
 from dask_expr.io import ReadParquet
 
+# Import backend DataFrame library to test
+BACKEND = config.get("dataframe.backend", "pandas")
+lib = importlib.import_module(BACKEND)
```

Comment:
@phofl - Any opinions on using this approach as a "switch" to test the `cudf` backend?

Reply:
Certainly interested in your thoughts as well @mrocklin

Reply:
I think this is ok, although it will take me some time to get used to it😂

traveling today, will take a closer look at the whole pr when I am back